src/lib/containers/user_profile_links/UserProfileLinks.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { useGetUserProfile } from '../../utils/hooks/SynapseAPI/useUserBundle'
+import Typography from '../../utils/typography/Typography'
+import IconSvg from '../IconSvg'
+import UserChallenges from './UserChallenges'
+import UserProjects from './UserProjects'
+import UserTeams from './UserTeams'
+
+export enum UserProfileLinksTabs {
+  PROJECTS = "Projects",
+  TEAMS = "Teams",
+  CHALLENGES = "Challenges",
+}
+
+type UserProfileLinksProps = {
+  userId: string
+}
+
+function UserProfileLinks({
+  userId,
+}: UserProfileLinksProps) {
+  const [currentTab, setCurrentTab] = useState<UserProfileLinksTabs>(UserProfileLinksTabs.PROJECTS)
+  const { data: userProfile } = useGetUserProfile(userId)
+
+  function getIconName(
+    currentTab: UserProfileLinksTabs,
+  ): string {
+    switch (currentTab) {
+      case UserProfileLinksTabs.PROJECTS:
+        return 'dashboard'
+      case UserProfileLinksTabs.TEAMS:
+        return 'peopleOutlined'
+      case UserProfileLinksTabs.CHALLENGES:
+        return 'challengesOutlined'
+    }
+  }
+  return (
+    <div className="UserProfileLinks">
+      <Typography variant="headline2" className="title">{userProfile?.userName}&apos;s Items</Typography>
```

Comment:
Just a really, _really_ picky, minor note: this could look weird if this renders before the user profile is fetched, if it's not cached in react-query. Maybe would be worth using a skeleton here.

Since this will be embedded in SWC, I'm not sure if a cache miss is likely; it might not make a difference--that probably depends on how we're populating the user cards